library/src/main/java/com/pokegoapi/api/pokemon/Pokemon.java

```diff
@@ -170,9 +172,12 @@ public Result transferPokemon() throws LoginFailedException, RemoteServerExcepti
 	 * @return the boolean
 	 * @throws LoginFailedException  the login failed exception
 	 * @throws RemoteServerException the remote server exception
+	 * @throws InvalidCurrencyException the currency has exception
 	 */
-	public boolean canPowerUp() throws LoginFailedException, RemoteServerException {
-		return getCandy() >= getCandyCostsForPowerup();
+	public boolean canPowerUp() throws LoginFailedException, RemoteServerException, InvalidCurrencyException {
+		return getCandy() >= getCandyCostsForPowerup()
```

Comment:
But it was an exception. I already fix it in last PR. 
@LoungeKatt pull last changes from Development and eveything should work without exceptions.